builds/serverless-config-pack/serverless-java.yml

```diff
@@ -112,5 +125,23 @@ resourcesDisabled:
         Name: {event_source_kinesis}
         ShardCount: 1
     #End:isStreamEnabled
+    #Start:isDynamoDbEnabled
+    #Start:isDynamoDbtableNotExist
+    sourceDynamoDbTable:
+       Type: AWS::DynamoDB::Table
+       Properties:
+         TableName: {event_source_dynamodb}
+         AttributeDefinitions:
+           - AttributeName: ID
```

Comment:
This seems to be a change in the requirement. Earlier the story was only to enable stream(if not already done). Now  I am seeing we are getting into creating the table(if it does not exist). The issue here is that partition key has to be given when creating it and it cannot be changed. You can add or delete secondary indexes. @devsatishm when did this requirement change and are we ok with this

Reply:
Good point. We are bringing in the ability to create tables with few (_hardcoded_) configurations to begin with. This will change later and developer is expected to provide relevant information (like participation key for example) during service creation through UI/CLI or using some kind of application template configurations.